docs/user/FlowVariables.md

```diff
@@ -0,0 +1,230 @@
+
+
+## Design Specific Configuration Variables
+
+
+### Required Variables
+
+
+Following minimum design variables must be defined in the design configuration
+file for each design located in the OpenROAD-flow-scripts directory of
+`./flow/designs/<PLATFORM>/<DESIGN_NAME>/config.mk`
+
+
+| Variable        | Description                                                                                                                                                          |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `PLATFORM`      | Specifies process design kit or technology node to be used.                                                                                                          |
+| `DESIGN_NAME`   | The name of the top-level module of the design.                                                                                                                      |
+| `VERILOG_FILES` | The path to the design Verilog files.                                                                                                                                |
+| `SDC_FILE`      | The path to design constraint (SDC) file.                                                                                                                            |
+| `CLOCK_PORT`    | The name of the designs clock port used in Static Timing Analysis. If your design does not have a clock port defined as an empty variable `export CLOCK_PORT= ""`. |
+| `CLOCK_PERIOD`  | The clock period for the design to meet its goal.                                                                                                                    |
+
+
+### Optional Variables
+
+
+These are optional variables that may be over-ridden/appended with
+default value from the platform `config.mk` by defining in the design
+configuration file.
+
+
+#### Synthesis
+
+
+| Variable                 | Description                                                                                        |
+|--------------------------|----------------------------------------------------------------------------------------------------|
+| `ADDITIONAL_LEFS`        | Hardened macro LEF view files listed here.                                                         |
+| `ADDITIONAL_LIBS`        | Hardened macro library files listed here.                                                          |
+| `ADDITIONAL_GDS`         | Hardened macro GDS files listed here.                                                              |
+| `VERILOG_INCLUDE_DIRS`   | Specifies the include directories for the Verilog input files.                                     |
+| `VERILOG_FILES_BLACKBOX` | Hardened macro Verilog file list to be added here.                                                 |

```

Comment:
This isn't an official flow variable. If it was it's deprecated and not used anywhere.